src/scenes/Document/scenes/Policy/Policy.js

```diff
@@ -30,9 +30,9 @@ function Policy({
       case ENCRYPT_STATES.PROTECTED:
         return null;
       case ENCRYPT_STATES.PROTECTED_NO_AUTH:
-        return <Button onClick={openAuthModal}>Sign In to Protect</Button>;
+        return <Button onClick={() => setAuthOpen(true)}>Sign In to Protect</Button>;
       default:
-        if (isLoggedIn) {
+        if (authState) {
```

Comment:
`authState` is boolean?

Reply:
ah, yes `authState = email && Virtru.Auth.isLoggedIn({ email }) ? 'loggedin' : false;`
gotta love "is"